Automattic/wp-calypso
# Add share menu to post card

This PR adds the `Share` action menu item to the post in the new post listing, and reveals the in-line sharing options. It also bumps a stat indicating that the menu item was used.

![screen shot 2017-08-14 at 11 29 30 pm](https://user-images.githubusercontent.com/1017839/29292774-b750a54a-8148-11e7-90b1-f8a6fa768265.png)

![share2](https://user-images.githubusercontent.com/1017839/29728308-acc07f50-89d7-11e7-8950-47ba1205fe70.gif)

This PR is part of a larger epic and behind the `posts/post-type-list feature flag`.
See p8F9tW-hL-p2.

To test:
- Check out this branch
- `ENABLE_FEATURES=posts/post-type-list npm start`
- Navigate to Blog Posts
- Click on the ellipsis menu for a post
- Click Share
- Verify that the Share panel is visible
- Verify that the Share panel can be closed by clicking the X
- Verify that the Share panel can be closed by selecting clicking the Share menu item
